providers/gce/gce_loadbalancer_internal_test.go

```diff
@@ -492,7 +520,7 @@ func TestUpdateInternalLoadBalancerNodes(t *testing.T) {
 	assert.Contains(
 		t,
 		instances[0].Instance,
-		fmt.Sprintf("%s/zones/%s/instances/%s", vals.ProjectID, vals.ZoneName, node2Name),
+		fmt.Sprintf("projects/%s/zones/%s/instances/%s", vals.ProjectID, vals.ZoneName, node2Name),
```

Comment:
Please don't change that. `legacy-cloud-provider` has too old version of the /api library, which has different behavior for listing instances.

This was fixed in https://github.com/kubernetes/cloud-provider-gcp/pull/262 for cloud-provider-gcp repository.